### PR TITLE
Add NativeLibrary support to README.md, remove now unneeded DllMap fi…

### PR DESCRIPTION
**Pull request by chmorgan**
…le and mono specific readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,6 +69,9 @@ SharpPcap usage.
 
 The examples are also a great resource a they show working examples using the latest API.
 
+* NativeLibrary is used for improved capture library resolution
+  * Improves library reosolution situation on Linux distros where there is a libpcap.so.X.Y symlink but no libpcap.so symlink
```
Comment (kayoub5) [inline README.md:73]: I still have not done this yet in #236, I just made it a lot more easier to implement in a future PR

File: README.md [hunk 2]
```diff
@@ -29,6 +29,10 @@ Note that packet dissection and creation was split from SharpPcap some years ago
   * Dumping packets to Pcap files.
   * Pcap and pcap-ng format (when using libpcap >=1.1.0 or npcap)
 
+* NativeLibrary support
+  * Capture library resolution works cleanly across Linux, OSX, and Windows
```
Comment (kayoub5) [inline README.md:33]: Please mention that it only works for .NET Core 3 and .NET Framework

**Comment by chmorgan** [comment]
@kayoub5 at the moment I'm thinking ping you on this one again once we have the support for the different libraries on linux

**Review by kayoub5 — APPROVED** [review]

**Comment by kayoub5** [comment]
@chmorgan any reason to hold merging this?

**Pull request merged**